CribblyBackend.UnitTests/ControllerTests/DivisionControllerTests.cs

```diff
@@ -18,10 +20,15 @@ public class DivisionControllerTests
         private readonly DivisionController divisionController;
         private readonly Mock<IDivisionService> mockDivisionService;
         private readonly Mock<ILogger> mockLoggerService;
+        private Team _team1 = new Team() { Id = 1, Name = "team1"};
+        private Team _team2 = new Team() { Id = 2, Name = "team2"};
+        private Division _division = new Division() { Id = 1, Name = "Test", Teams = new List<Team>()};
```

Comment:
🌵 📖 in .NET5 (once my PR is merged), you can do:
```
private Division _division = new() {...};
```

Reply:
What exactly do you mean by this?

Reply:
You don't have to specify the type you're `new`ing up if you've declared it on the lefthand side of the assignment.

Reply:
Gotcha, that's neat. Fixed it!